00/ht/Space647/answers.md

```diff
@@ -0,0 +1,160 @@
+5. undefined    var x;
+6. let name = Symbol.for("name");
+alert( Symbol.for("name") == name );
+```
+# 5.	Как узнать тип переменной? Пример кода
+```javascript 
+1. number var i=7.4151514 
+console.log(typeof(i)); 
+2. string var s=”Dog”
+console.log(typeof(s)); 
+3. null var s=null
+console.log(typeof(s)); 
+4. Boolean (true false) var t=true; f=false;
+console.log(typeof(t,f)); 
+5. undefined    var x;
+console.log(typeof(x));
+6. let sym = Symbol("name");
+alert( sym.toString() );
+7. object var name={name:”Gleb”};
+console.log(typeof(name)); 
+8.  function
+Console.log(typeof(function(){}));`
+```
+# 6.	что такое область видимости?
+Область видимости-существует 2 вида переменных локальные и публичные, публичные доступны везде, локальные доступны только в текущей области видимости
+# 7.	как из javascript в брaузере вывести на экран строчку ? ( 4-ре способа ) Примеры кода
+```javascript 
+1.	alert(“First”);
+2.	prompt(“second”,2);
+3.	confirm(“third”)
+4.	console.log(“Fourth”);
+5.	document.write(“Fifth”);
+6.	div.innerHTML=”sixth”;
+```
+# 8.	каким образом можно реализовать условное выполнение кода ?
+при помощи if/else и :?
+# 9.	условные операторы в javascript - какие? Примеры использования
+`if/else и :?
+
+If(i>0)
+{
+Console.log(true);
+}
+Else
+{
+Console.log(false);
+}
+
+bigAge=(age>14? True:false);`
+
+# 10.	что такое тернарный оператор?
+?:   замена if/else которая может принимать три операнда.
+условие ? выражение1 : выражение2
+# 11.	Как обойтись без условных операторов если нужно сделать условное выполнение кода? С примером
+При помощи тернарный операнда
+```javascript 
+        var Name = prompt('Ваше имя?');
+        var Otchestvo = prompt('Ваше отчество?');
+        var SecondName = prompt('Ваша фамилия?');
+        var age = prompt('Вашь возраст (в годах)');
+        var male = confirm("Вы - мужчина?");
+        var days = age * 365;
+        (male==true)?(age>=60? alert('ФИО:'+Name+' '+Otchestvo+' '+SecondName+'\n' +'Возраст,лет '+age+'\n'+'Возраст дней '+days+'\n'+'Пол Мужской'+'\n'+'Пенсионер'):alert('ФИО:'+Name+' '+Otchestvo+' '+SecondName+'\n' +'Возраст,лет '+age+'\n'+'Возраст дней '+days+'\n'+'Пол Мужской'+'\n'+'не пенсионер')):(age>=55?alert('ФИО:'+Name+' '+Otchestvo+' '+SecondName+'\n' +'Возраст,лет '+age+'\n'+'Возраст дней '+days+'\n'+'Пол женский'+'\n'+'Пенсионер'):alert('ФИО:'+Name+' '+Otchestvo+' '+SecondName+'\n' +'Возраст,лет '+age+'\n'+'Возраст дней '+days+'\n'+'Пол женский'+'\n'+'Пенсионер'));`
+ ```
+
+# 12.	что такое функция? какими характеристиками можно описать функцию?
+функция –Это участок кода который можно использовать многократно. 
+это упроститель жизни)) В функцию в которой есть входной параметр, можно передать значения для расчетов цены апельсинов. Многократное использование.
+# 13.	Как можно создать функцию? 3 способа. C примерами кода
+```javascript 
+1.	new Function
+var gleb= new function(){};`
+2.	Декларативное создание функции
+Function gleb([111]){};`
```

Comment:
![https://www.dropbox.com/s/hrenjmc9rs00hqi/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202017-05-15%2020.05.38.jpg?dl=1](https://www.dropbox.com/s/hrenjmc9rs00hqi/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202017-05-15%2020.05.38.jpg?dl=1)